Apps/Examples/Examples/Models/Examples.swift

```diff
@@ -155,9 +112,6 @@ public struct Examples {
         Example(title: "Display buildings in 3D",
                 description: "Use extrusions to display buildings' height in 3D.",
                 type: BuildingExtrusionsExample.self),
-        Example(title: "Use OfflineRegionManager to download a region",
-                description: "Use the deprecated OfflineRegionManager to download regions for offline use.",
-                type: OfflineRegionManagerExample.self),
```

Comment:
Why is this deleted?

Reply:
It was generating deprecation warnings, which failed the build because I reconfigured the Examples app to use the shared base.xcconfig which treats warnings as errors. Since the API is deprecated, we're still in beta, and there's a [note](https://github.com/mapbox/mapbox-maps-ios/pull/187/files#diff-df75f4ee000e948ae046c8e408d7bc38a1750908a947bdf531f3274f687d0d3eL52) warning that it will be removed in a future release, I decided the easiest path was to delete it.

I realized later that I could override the warnings as errors for the Release config using the Examples_testhost.xcconfig file, so if we do need to keep it, I'll update the PR to do that instead.

Reply:
Let's keep it if we can until there's an alternative.